infra/tpu-pytorch-releases/artifacts_builds.tf

```diff
@@ -95,6 +118,53 @@ module "nightly_builds" {
   docker_repo_url         = module.docker_registry.url
 }
 
+// TODO: Remove this after the 2.1 release
+module "xrt_nightly_builds" {
+  source   = "../terraform_modules/xla_docker_build"
+  for_each = local.xrt_nightly_builds_dict
+
+  ansible_vars = merge(each.value, {
+    package_version = var.nightly_package_version
+    nightly_release = true
+    pytorch_git_rev = "main"
+    xla_git_rev     = "$COMMIT_SHA"
+  })
+
+  trigger_on_schedule = { schedule = "0 0 * * *", branch = "xrt" }
+
+  trigger_name = "nightly-xrt-${replace(each.key, "/[_.]/", "-")}"
+  image_name   = "xla"
+  image_tags = [
+    "nightly_xrt_${each.key}",
+    # Append _YYYYMMDD suffix to nightly image name.
+    "nightly_xrt_${each.key}_$(date +%Y%m%d)",
+  ]
+
+  description = join(" ", [
+    "Builds nightly xla:nightly_${each.key}' ${
+      each.value.accelerator == "tpu"
+      ? "TPU"
+      : format("CUDA %s", each.value.cuda_version)
+    } docker image and corresponding wheels for PyTorch/XLA.",
+    "Trigger managed by Terraform setup in",
+    "infra/tpu-pytorch-releases/artifacts_builds.tf."
+  ])
+
+  wheels_dest = "${module.releases_storage_bucket.url}/wheels/xrt/${
```

Comment:
Very good, the `xrt` directory is needed, as these wheels will have the same name as non-xrt wheels.